content/posts/newsletter/2020-02-04.md

```diff
@@ -0,0 +1,264 @@
+
+![Dino](https://dino.im/blog/2020/01/dino-0.1-release/main_window.png)
+
+Tigase has released [BeagleIM 3.5 and SiskinIM 5.5](https://tigase.net/beagleim-3.5-and-siskin-5.5-released/), respectively for macOS and iOS.
+
+![Beagle IM](https://tigase.net/static/28fe03d4f74066874cfcf0ca8f5b7732/fcda8/beagle-chat.png)
+
+JC Brand has announced the release of [Converse.js 6.0.0](https://github.com/conversejs/converse.js/releases/tag/v6.0.0).
+
+Georg Lukas has released  [yaxim 0.9.9 - FOSDEM Edition](http://yaxim.org/blog/2020/01/31/yaxim-0-dot-9-9-fosdem-edition/).
+
+In the latest newsletter, we wrongly announced the release of profanity 0.8.0 due to a misunderstanding. Please be aware that [profanity 0.8.0 has been released](https://github.com/profanity-im/profanity/releases/tag/0.8.0) on the 3rd of February. Also, [a new blog has opened](https://profanity-im.github.io/blog/index.html).
+
+Monal has seen [a lot of news in January](https://monal.im/2020/01/), among which Catalyst progress, iOS [4.2.1](https://monal.im/blog/monal-ios-4-2-1-released/) and [4.2.2](https://monal.im/blog/catalyst-and-ios-4-2-2/), and 4.2.3 releases to Testflight, [UI progress](https://monal.im/blog/mam-and-rethinking-the-ui/) in upcoming 4.3.
+
+Please also follow progress:
+
+* [Gajim development news January 2020](https://gajim.org/post/2020-01-27-development-news-january/)
+* [SàT progress note 2020-W01](https://www.goffi.org/b/66adNNNbFp84CWFzyvGpmK/progress-note)
+* zatrox, jbb & melvo, from the Kaidan team, explain their upcoming [streamlined registration process](https://www.kaidan.im/2020/01/08/Easy-Registration/) in the next Kaidan 0.5.0.
+
+### Libraries 
+
+Remi Corniere has released [go-xmpp v0.4.0](https://blog.process-one.net/go-xmpp-v0-4-0/).
+
+Tigase has just introduced [halcyon](https://github.com/tigase/halcyon), a Kotlin multiplatform XMPP client library.
+
+## Other 
+
+Mastodon is getting [support for XMPP URIs](https://github.com/tootsuite/mastodon/issues/9776).
+
+The EU H2020 [CPSwarm](https://www.cpswarm.eu/) project has produced as one of its main outcome the [CPSwarm Workbench](https://www.cpswarm.eu/index.php/cpswarm-workbench/), a toolchain to model, optimize, simulate and deploy a swarm of Cyber Physical Systems. One of its opensource components [released on Github](https://github.com/cpswarm), is the [Simulation and Optimization Environment](https://github.com/cpswarm/SimulationOrchestrator/wiki/Simulation-and-Optimization-Environment), which is used to evaluate the performance of a swarm solution. This environment has a distributed architecture based on XMPP, allowing to execute in parallel simulations on simulators deployed on distributed machines.
+
+## Services 
+
+Tigase has upgraded their public servers [xmpp.cloud](https://xmpp.cloud) with SASL-EXTERNAL for federation, and MUC avatars.
```

Comment:
Where was this announced?

Reply:
@licaon-kter, here I suppose: <https://mastodon.technology/@tigase/103488503676998610>

Reply:
Maybe link to that too?